dev-libs/libmspack/libmspack-0.5alpha.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="A portable library for Microsoft compression formats"
+DESCRIPTION="The following file formats are supported: \
```

Comment:
You have a Control-V just before the word "The" in DESCRIPTION.
(Can you remove that unprintable char?)